src/ansible_navigator/app.py

```diff
@@ -78,46 +84,67 @@ def app(self) -> AppPublic:
         raise AttributeError("app passed without args initialized")
 
     @staticmethod
-    def _copy_args(args):
-        """deepcopy the args, but un mount the cache first
+    def _copy_args(args: ApplicationConfiguration) -> ApplicationConfiguration:
+        """Deepcopy the args.
+
+        Note: Unmount the collection doc cache (cdc) first
         the cdc will get mounted if the child needs it
         in parse and update
```

Comment:
Let's improve the RST rendering a bit:
```suggestion
        .. note::

           Unmount the collection doc cache (cdc) first
           the cdc will get mounted if the child needs it
           in parse and update
```